summoner-cli/src/Summoner/Template/GitHub.hs

```diff
@@ -294,12 +360,15 @@ gitHubFiles Settings{..} = concat
     cabalUpdate :: Text
     cabalUpdate = "cabal v2-update"
 
+    cabalConfigure :: Text
+    cabalConfigure = "cabal v2-configure --enable-tests --enable-benchmarks"
+
     cabalBuild :: Text
-    cabalBuild = "cabal v2-build --enable-tests --enable-benchmarks"
+    cabalBuild = "cabal v2-build all"
 
     cabalTest :: Text
     cabalTest = if settingsTest
-        then "cabal v2-test --enable-tests --test-show-details=direct"
```

Comment:
This flags we implemented under #466, and the proper fix is to bump up Cabal version on CI, not delete flags. The `--test-show-details=direct` flag should go to the `configure` command

Reply:
Ok, I will bump Cabal version and restore the flags

Reply:
Flag restored, Cabal bumped to 3.0.